src/main/scala/org/scalajs/tools/tsimporter/Importer.scala

```diff
@@ -152,13 +152,15 @@ class Importer(val output: java.io.PrintWriter) {
         processDefDecl(classSym, Name.CONSTRUCTOR,
             FunSignature(Nil, params, Some(TypeRefTree(CoreType("void")))), Set.empty[Modifier])
 
-      case PropertyMember(PropertyNameName(name), opt, tpe, mods) if mods(Modifier.Static) =>
+      case PropertyMember(PropertyNameName(name), opt, underlying, mods) if mods(Modifier.Static) =>
```

Comment:
Please keep `tpe` in the pattern, as it is the name of the field in `PropertyMember`.

Reply:
https://github.com/sjrd/scala-js-ts-importer/pull/105/commits/c6a56b1986f1fca34f9e33ab4afbc0826d808f8b